.travis.yml

```diff
@@ -24,11 +25,15 @@ deploy:
   domain: material.preactjs.com
   skip_cleanup: true
   on:
-    branch: master
+    branch: release
     condition: $(./scripts/has-changes.sh docs) = "yes" && $TEST_DIR = "docs"
     node: "8"
 
 # blocklist
 branches:
   except:
-    - gh-pages
\ No newline at end of file
+    - gh-pages
+
+sudo: required
+addons:
+ chrome: stable
```

Comment:
Is this necessary? `puppeteer` should install chrome by itself.

Reply:
nups without this, it doesnt work